parent/pom.xml

```diff
@@ -351,6 +357,12 @@
         <version>2.8.7</version>
       </dependency>
 
+      <dependency>
+        <groupId>org.codehaus.plexus</groupId>
+        <artifactId>plexus-interpolation</artifactId>
```

Comment:
Do you know why this dependency is required ?

Reply:
It's being used here: https://github.com/fabric8io/fabric8-maven-plugin/blob/7d7b13e3f675543d09e56eccc71f20abd42be0ae/core/src/main/java/io/fabric8/maven/core/service/openshift/OpenshiftBuildService.java#L426